magenta/music/midi_io.py

```diff
@@ -293,6 +297,12 @@ def note_sequence_to_pretty_midi(
     pm._update_tick_to_time(0)
     # pylint: enable=protected-access
 
+  # Populate instrument names by first creating an instrument map between instrument index and name
+  # Then going over this map in the instrument event for loop
+  instrument_names = {}
+  for inst_name in sequence.instrument_infos:
```

Comment:
s/inst_name/instrument_info/

Reply:
I changed `instrument_names` to `inst_infos`, and `inst_name` to `inst_info`. I hope that what you meant.